apiary.apib

```diff
@@ -5660,6 +5660,90 @@ independently on the configurations.
                 "changeDescription": "change description"
             }
 
+### Update Row State [PUT /v2/storage/components/{component_id}/configs/{config_id}/rows/{row_id}/state]
+Updates state of an existing row.
+
++ Parameters
+    + component_id (required) - Id of the component - e.g. `keboola.ex-db-mysql`
+    + config_id (required) - Id of the configuration
+    + row_id (required) - Id of the configuration row
+
++ Attributes
+    + state (required) - Serialized configuration state JSON; the maximum allowed size is 4MB
+
++ Request (application/x-www-form-urlencoded)
+    + Headers
+
+            X-StorageApi-Token: your_token
+
+    + Body
+
+            state%3D%7B%22lastId%22%3A123%7D
+
++ Response 200 (application/json)
+
+            {
+                "id": "236781529",
+                "configuration": {
+                    "host": "localhost",
+                    "port": null
+                },
+                "state": {},
```

Comment:
tu by v tej ukazke by mohlo byt ten state co mas v tom body v ukazke requestu

Reply:
Upravil jsem to i pro konfigurace.